packages/http-client-python/eng/pipeline/publish.yml

```diff
@@ -30,3 +30,8 @@ extends:
           LanguageShortName: "python"
           CadlRanchName: "@typespec/http-client-python"
           EnableCadlRanchReport: false
+      - script:
+          pnpm chronus version --prerelease --only @typespec/http-client-python
+          git add $(Build.SourcesDirectory)/packages/http-client-python
```

Comment:
is that to reset the other packages?
can you add a comment linking to the bug https://github.com/timotheeguerin/chronus/issues/359

Reply:
Hi @timotheeguerin, as PR title said, we just want to release dev version for python once there is PR merged into main branch. Does https://github.com/timotheeguerin/chronus/issues/359 block the PR? If yes, appreciate if you could guide us how to add proper script. Thanks!

Reply:
no, this is a workaround which is fine